src/main/java/nurseybook/model/NurseyBook.java

```diff
@@ -177,6 +177,14 @@ public void addPossibleGhostTasksWithMatchingDate(LocalDate keyDate) {
         tasks.addPossibleGhostTasksWithMatchingDate(keyDate);
     }
 
+    public void updateTasksOverdueStatus() {
+        tasks.updateOverdueStatuses();
+    }
+
+    public void updateRecurringTasksDate() {
```

Comment:
Similarly, a javadoc here as well would be good. But if it seems rather repetitive as there are similar methods in other files (that are not overriding this) with the same functionality, you can choose not to do so.

Reply:
Alright! I think the current code is pretty lacking in documentation I'll see to rectifying it